server/GRPC_TA2_TA3/TA3.py

```diff
@@ -0,0 +1,149 @@
+import grpc
+import core_pb2
+import core_pb2_grpc
+from protobuf_to_dict import protobuf_to_dict, dict_to_protobuf # this library to converts python grpc messages to dict
+from enum import Enum
+from tornado import websocket, web, ioloop
+import json
+
+channel = grpc.insecure_channel('localhost:50051')
+coreStub = core_pb2_grpc.CoreStub(channel)
+
+
+
+MESSAGE_TYPE = {
+	'BLOCKING': 'BLOCKING',
+	'STREAMING': 'STREAMING'
+}
+
+
+grpcCall = {
+	# rpc CreatePipelines(PipelineCreateRequest) returns (stream PipelineCreateResult) {}
+	'CreatePipelines': 
+		{
+			'function': coreStub.CreatePipelines,
```

Comment:
It looks like the file has mixed spaces/tabs. Can we use 2- or 4-space everywhere? Thanks.